integration/init_test.go

```diff
@@ -103,6 +103,7 @@ func TestIntegration(t *testing.T) {
 	SetDefaultEventuallyTimeout(30 * time.Second)
 
 	suite := spec.New("Integration", spec.Report(report.Terminal{}))
-	suite("Default", testDefault, spec.Parallel())
```

Comment:
I see we removed the `spec.Parallel()` - was that intentional? Did you observe the tests failing with that present?